packages/wrangler/src/sites.tsx

```diff
@@ -120,7 +115,9 @@ export async function syncAssets(
     return { manifest: undefined, namespace: undefined };
   }
 
-  const title = `__${scriptName}_sites_assets${preview ? "_preview" : ""}`;
+  const title = `__${scriptName}${env ? `-${env}` : ""}-workers_sites_assets${
+    preview ? "_preview" : ""
+  }`;
```

Comment:
This is the change to bring the title inline with Wrangler1?

Reply:
Correct, yes. Specifically, `s/_sites_assets/-workers_sites_assets`.